modules/social_features/social_search/social_search.install

```diff
@@ -323,3 +322,33 @@ function social_search_update_11403() : void {
     $config->save(TRUE);
   }
 }
+
+/**
+ * Revert search_api_language filter name/id change for groups view.
+ */
+function social_search_update_11404() : void {
+  // There was a typo in the view name of social_search_update_11403 which
+  // caused the groups view not to be updated, so we must do it again here.
+  $views = [
+    "views.view.search_groups",
+  ];
+
+  foreach ($views as $view) {
+    $config = \Drupal::configFactory()->getEditable($view);
+    $display = $config->get("display");
+
+    $filter = $display['default']['display_options']['filters']['language_with_fallback'];
+    $filter['id'] = "search_api_language";
+    $display['default']['display_options']['filters']["search_api_language"] = $filter;
+    unset($display['default']['display_options']['filters']['language_with_fallback']);
+
+    $config->set('display', $display);
+    $config->save(TRUE);
+  }
+
+  // Clean up the config created in our erroneous version of 11403.
+  $incorrect_view = \Drupal::configFactory()->getEditable("views.view.search_group");
+  if (!$incorrect_view->isNew()) {
```

Comment:
Thinking out loud here: https://www.drupal.org/project/social/issues/3383079
It seems some users potentially already deleted the view, we might want to check if it still exists as well (?)

Reply:
Ronald, when the configuration object does not exists, `configFactory` will create a new one thus marking it as "new", since the original code is already checking it for "isNew" I don't think that we need any extra checks.

![image](https://github.com/goalgorilla/open_social/assets/11061892/222ff849-c6ca-4e8a-98e9-bd0226949952)

![image](https://github.com/goalgorilla/open_social/assets/11061892/59db6f0d-5198-4a94-a107-5c2e107492db)

![image](https://github.com/goalgorilla/open_social/assets/11061892/4dd24fd6-672d-4a50-b80a-17283d471069)